crates/bevy_ecs/src/observer/trigger_event.rs

```diff
@@ -86,7 +92,7 @@ fn trigger_event<E, Targets: TriggerTargets>(
 ///
 /// [`Trigger`]: crate::observer::Trigger
 /// [`Observer`]: crate::observer::Observer
-pub trait TriggerTargets: Send + Sync + 'static {
+pub trait TriggerTargets: Send + Sync {
```

Comment:
Is there any reason not to move Send and Sync out too? I think by the same logic it shouldn’t be included on the trait unless necessary. This could make implementing it easier for reusing buffers.

Reply:
Thanks :)